### ISSUE TITLE
Compiling from release tarball fails due to miniupnpc-static

### ISSUE THREAD

**Issue by sammy007** [monero-project/monero#3859]
```
-- Found MiniUPnPc: /usr/include/miniupnpc
-- Found miniupnpc API version 10
-- Using in-tree miniupnpc
CMake Error at external/CMakeLists.txt:42 (add_subdirectory):
  add_subdirectory given source "miniupnp/miniupnpc" which is not an existing
  directory.


CMake Error at external/CMakeLists.txt:44 (set_property):
  set_property could not find TARGET libminiupnpc-static.  Perhaps it has not
  yet been created.


CMake Error at external/CMakeLists.txt:48 (set_property):
  set_property could not find TARGET libminiupnpc-static.  Perhaps it has not
  yet been created.
```

Happened with 0.12.1.0 point release.

```
dpkg -l| grep upnpc
ii  libminiupnpc-dev                       1.9.20140610-2ubuntu2.16.04.2              amd64        UPnP IGD client lightweight library development files
ii  libminiupnpc10:amd64                   1.9.20140610-2ubuntu2.16.04.2              amd64        UPnP IGD client lightweight library
ii  miniupnpc                              1.9.20140610-2ubuntu2.16.04.2              amd64        UPnP IGD client lightweight library client
```

Ubuntu 16.04.

**Comment by sammy007** [comment]
I understand that github's stupid tarball releases does not contain submodules, but why not fallback to miniupnpc from the system?

**Comment by sammy007** [comment]
@moneromooo-monero ah. Imo makes sense to get rid of auto tarballs then since they are poorly implemented.

**Comment by hyc** [comment]
and on older OSs the static libs aren't compiled -fPIC, so they can't be used anyway.

**Comment by anonimal** [comment]
>why not fallback to miniupnpc from the system?

See https://github.com/monero-project/monero/issues/3862#issuecomment-392183195, and just do a recursive git clone / check out tag. The repo is a development repo so git should be easily attainable. If people don't want to do development then they can simply use the release binaries.

>Imo makes sense to get rid of auto tarballs then since they are poorly implemented.

Yes. There should be a github/github ticket somewhere for this.

**Comment by ilovezfs** [comment]
The automatically available tarballs are just a thin wrapper around the `git archive` command. They cannot be disabled. If you want the submodules available in a source tarball, a separate source release tarball can be manually uploaded to the release in the GUI, which is pretty common. For example, see

https://github.com/zyedidia/micro/releases/tag/v1.4.0
and the attached
https://github.com/zyedidia/micro/releases/download/v1.4.0/micro-1.4.0-src.tar.gz

It's also perfectly valid to expect people to use the Git clone, although a complete source tarball can help with preventing issues such as the current one and is a nice convenience.

**Comment by moneromooo-monero** [comment]
Fixed in 0.13.0.4
